ThePortugueseMan.Calculator/ThePortugueseMan.Calculator/Program.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System.Linq;
+using CalculatorLibrary;
+
+namespace CalculatorProgram
```

Comment:
not mandatory but you can use the newest namespaces to optimize use of space: 

![image](https://user-images.githubusercontent.com/108711476/218612889-9978b84b-b957-4550-8bcb-a4969e166662.png)

Reply:
Done!